utils/clean.py

```diff
@@ -97,5 +104,215 @@ def clean_state(self) -> (pd.DataFrame, pd.DataFrame, pd.DataFrame):
+            df_list.append(pd.read_csv(filepath))
+
+        return df_list
+
+    def clean_state(filepaths: list[str]) -> (pd.DataFrame, pd.DataFrame, pd.DataFrame):
+        """Calls the other methods in order
+
+        This is the master function of the ArizonaCleaner
+        class, and calling it will activate the cleaning
+        pipeline which takes in filenames and outputs cleaned,
+        standardized, and schema-compliant tables
+
+        args: list of two filepaths which lead to dataframes
+
+        returns: three schema-compliant tables for
+        transactions, individuals, and organizations
+
+        """
+
+        transactions, details = ArizonaCleaner.preprocess(filepaths)
+
+        # employer = transactions.groupby("CommitteeId")[
+        #     "TransactionEmployer"].apply(lambda x: max(
+        #         x, key=lambda y: y if y is not None else "")).values
+        # #is only going to select one emplyer
+
+        # details["company"] = employer
+
+        cleaned_transactions, cleaned_details = ArizonaCleaner.clean(
+            [transactions, details]
+        )
+
+        standardized_transactions, standardized_details = ArizonaCleaner.standardize(
+            [cleaned_transactions, cleaned_details]
+        )
+
+        (
+            az_transactions,
+            az_individuals,
+            az_organizations,
+        ) = ArizonaCleaner.create_tables(
+            [standardized_transactions, standardized_details]
+        )
+
+        return (az_transactions, az_individuals, az_organizations)
+
+    def create_tables(
+        data: list[pd.DataFrame],
+    ) -> (pd.DataFrame, pd.DataFrame, pd.DataFrame):
+        """split up cleaned and standardized tables to undergo final formatting
+
+        We split up the details tables into individuals (individual contributors
+        and donors) and organizations (everything else), which then undergo
+        conversion into fully schema-compatible transactions, individual details
+        and organization details tables.
+
+        args: cleaned and standardized transactions and details tables
+
+        returns: three tables for transactions,
+        individual details, and organization details
+
+        """
+
+        transactions, details = data[0], data[1]
+
+        individual_details = details[
+            (details["entity_type"] == "Individual")
+            | (details["entity_type"] == "Candidate")
+        ]
+        organization_details = details[
+            (details["entity_type"] != "Individual")
+            & (details["entity_type"] != "Candidate")
+        ]
+
+        # individual_details = cs_details[
+        #     cs_details["entity_type"] == ("Individual Contributors" or "Candidate")
+        # ]
+
+        # organization_details = cs_details[
+        #     cs_details["entity_type"] != ("Individual Contributors" or "Candidate")
+        # ]
+
+        az_transactions = az_transactions_convert(transactions)
```

Comment:
Can the utils functions here go inside of the abstract class or are they used in the web scraper also?